airflow/providers/microsoft/azure/CHANGELOG.rst

```diff
@@ -18,6 +18,13 @@
 
 Changelog
 ---------
+2.0.1
+.....
+
+Bug fixes
+~~~~~~~~~
+
+* ``BugFix: Fix remote log in azure storage blob when blob already exists (#15907)``
```

Comment:
You don't need to add the changelog. Providers changelogs are generated automatically when they are to be released

Reply:
Lol, will fix that too